components/tools/OmeroWeb/omeroweb/webclient/show.py

```diff
@@ -896,6 +846,87 @@ def paths_to_object(conn, experimenter_id=None, project_id=None,
     return paths
 
 
+def paths_to_well_image(conn, params, well_id=None, image_id=None,
+                        acquisition_id=None,
+                        plate_id=None, screen_id=None, experimenter_id=None,
+                        orphanedImage=False):
+
+    qs = conn.getQueryService()
+    service_opts = deepcopy(conn.SERVICE_OPTS)
+    q = '''
+        select coalesce(sowner.id, plowner.id, aowner.id, wsowner.id),
+               slink.parent.id,
+               plate.id,
+               acquisition.id,
+               well.id
+        from WellSample wellsample
+        left outer join wellsample.details.owner wsowner
+        left outer join wellsample.plateAcquisition acquisition
+        left outer join wellsample.details.owner aowner
+        join wellsample.well well
+        left outer join well.plate plate
```

Comment:
`plate` is a required element of `well` as is `details` as above so do not need to be `left outer` joined.